src/renderer/kube-object.store.ts

```diff
@@ -149,8 +149,10 @@ export abstract class KubeObjectStore<T extends KubeObject = any> extends ItemSt
       if (merge) {
         this.mergeItems(items, { replace: false });
       } else {
-        return items;
+        this.mergeItems(items, { replace: true });
```

Comment:
Why `replace: true`? 
This way random extension can screw up loaded items with `loadAll({namespaces: ["default"]})`?

Reply:
They can only screw up if they explicitly set `merge: false`, right? Maybe I don't understand what you originally meant with `merge` option. Anyway it's not obvious for a user of this function if `merge: false` does not update the store. 

That said: I can revert this change if needed but the watch api reload needs a way to force a fresh items array.

Reply:
Yeah, right. Since by default `merge = true`